command-snapshot.json

```diff
@@ -27,8 +27,8 @@
     "alias": ["force:org:snapshot:delete"],
     "command": "org:delete:snapshot",
     "flagAliases": ["apiversion", "targetdevhubusername"],
-    "flagChars": ["s", "v"],
-    "flags": ["api-version", "flags-dir", "json", "loglevel", "snapshot", "target-dev-hub"],
+    "flagChars": ["p", "s", "v"],
+    "flags": ["api-version", "flags-dir", "json", "loglevel", "no-prompt", "snapshot", "target-dev-hub"],
```

Comment:
Confirming, `(-p|--no-prompt)` is a well-known pattern for `sf` tool actions?

Reply:
yeah, most commands do `-p/--no-prompt`:
https://github.com/search?q=org%3Asalesforcecli+%2F%27no-prompt%27%2F&type=code